content/developer/reference/backend/orm/changelog.rst

```diff
@@ -13,10 +44,8 @@ Odoo Online version 15.4
   and are less confusing about what is actually done in each case.
   See the section :ref:`SQL Execution <reference/orm/sql>`.
 
-Odoo Online version 15.2
+Odoo Online version 15.5
```

Comment:
I wouldn't reverse the order of versions. In a few years, it will be better to have the latest versions on top, and `saas 15.2` at the bottom.

Reply:
Yep, it seems logical to me. Updated!